docs/rules/lines-around-comment.md

```diff
@@ -48,123 +46,114 @@ Empty lines are also not required at the beginning or end of a file.
 
 ## Options
 
-This rule has 10 options.
+This rule has an object option:
 
-2 options for block comments:
+* `"beforeBlockComment": true` (default) enforces an empty line before block comments
```

Comment:
What are the pros and cons of enforces compared to requires in description of options?

It seems like enforce consistent style or use in rule description often corresponds to options to disallow or require.